web/forms.py

```
@@ -1802,3 +1804,70 @@ class StudyGroupForm(forms.ModelForm):
+                work_type = WorkType.objects.get(pk=work_type_id)
+                allowed_extensions = work_type.get_allowed_extensions_list()
+                accept_types = ",".join([f".{ext}" for ext in allowed_extensions])
+                self.fields["work_file"].widget.attrs["accept"] = accept_types
+                self.fields["work_file"].help_text = (
+                    f"Upload your document (max {work_type.max_file_size_mb}MB, "
+                    f"allowed types: {', '.join(allowed_extensions)})"
+                )
+            except (ValueError, WorkType.DoesNotExist):
+                pass
+
+    def clean_work_file(self):
```

Ruff (0.8.2)

Missing return type annotation for private function `clean_work_file`

(ANN202)